crates/nu-command/src/bits/not.rs

```diff
@@ -90,7 +90,7 @@ impl Command for SubCommand {
             Example {
                 description:
                     "Apply logical negation to a list of numbers, treat input as 2 bytes number",
-                example: "[4 3 2] | bits not -n 2",
+                example: "[4 3 2] | bits not -n '2'",
```

Comment:
I'm confused by several of these changes. This one, for example, "treat input as 2 bytes number". Wouldn't it be better to change the SyntaxShape to Int instead of changing a number to look like a string?

Reply:
@fdncred - It's not actually a number, that confused me to. The description reads:

```
  -n, --number-bytes <String> - the size of unsigned number in bytes, it can be 1, 2, 4, 8, auto
```

Reply:
Right, but my point was that intead of changing it to `-n '2'` we should change the SyntaxShape to Int instead of leaving it as a String.

Reply:
Agree with you now - i totally missed the `auto`